### PR TITLE
make interop conditional more clear

### PR DESCRIPTION
**Pull request by hassanbazzi**

### CODE REVIEW

File: src/lib/entry.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ else if ('serviceWorker' in navigator && location.protocol === 'https:') {
 }
 
 
-const interopDefault = m => m && m.default || m;
+const interopDefault = m => m && m.default ? m.default : m;
```
Comment (reznord) [inline src/lib/entry.js]: This is easy to understand.

👍 LGTM!

**Comment by thangngoc89** [comment]
The way I see it is extractly the same. This is actually common in JS codebases. Try jQuery :D

File: src/lib/entry.js (hunk 1 above)
Comment (developit) [inline src/lib/entry.js]: but it takes more space :(

**Comment by developit** [comment]
heh yeah they look the same to me

File: src/lib/entry.js (hunk 1 above)
Comment (reznord) [inline src/lib/entry.js]: true, but easy to understand ;)

**Comment by lukeed** [comment]
I'm a fan of the `&&||` 😉 so i must be one too

**Comment by developit** [comment]
we are mutants

**Comment by lukeed** [comment]
or simulations 💻

**Comment by reznord** [comment]
Count me in !! ;)

**Pull request merged**